apps/api/src/app/workflows-v1/notification-template.controller.ts

```diff
@@ -107,8 +113,10 @@ export class NotificationTemplateController {
         tags: body.tags,
         description: body.description,
         workflowId: body.identifier,
-        critical: body.critical,
-        preferenceSettings: body.preferenceSettings,
+        defaultPreferences: DEFAULT_WORKFLOW_PREFERENCES,
+        userPreferences:
+          body.preferenceSettings &&
+          buildWorkflowPreferencesFromPreferenceChannels(body.critical ?? false, body.preferenceSettings),
```

Comment:
Internally, all Workflow DTOs now carry a `defaultPreferences` and `userPreferences` object now.